lib/pubnub/client.rb

```diff
@@ -382,7 +382,7 @@ def setup_app(options)
       Pubnub.logger = options[:logger] || Logger.new('pubnub.log')
       Concurrent.global_logger = Pubnub.logger
       @subscriber = Subscriber.new(self)
-      options[:user_id] = options[:uuid] until options[:user_id].nil?
+      options[:user_id] = options[:uuid] if options[:user_id].nil?
```

Comment:
@kleewho it was better to just replace to `unless`. Now condition is wrong, because it will reset `user_id` if `uuid` is nil.

Reply:
It's not wrong. It's good. It will set `user_id` to whatever value `uuid` is only if `user_id` is not set. That's what I wanted to achieve